packages/map-template/src/App.jsx

```diff
@@ -18,7 +18,7 @@ const mapboxAccessToken = params.get('mapboxAccessToken');
 
 // Append the hashtag symbol to the color code (i.e. ffffff)
 const primaryColor = params.get('primaryColor');
-const hexPrimaryColor = primaryColor ? '#'.concat(primaryColor) : undefined;
+const hexPrimaryColor = primaryColor ? '#'.concat(primaryColor) : '#005655';
```

Comment:
We still don't have a way to use MIDT in JavaScript?
If not, write a comment about which MIDT color this is.

Reply:
Added comment about HEX value.